test/support/api/program-actions.js

```diff
@@ -34,6 +34,7 @@ Cypress.Commands.add('routeProgramActions', (mutator = _.identity, programId) =>
     action.relationships = getProgramActionRelationships({
       program,
       owner: _.random(1) ? _.sample(fxTestTeams) : null,
+      programFlow: _.sample(fxProgramFlows),
```

Comment:
This wasn't causing failures in the tests, but it was causing this:

<img width="748" alt="Screenshot 2023-10-17 at 5 01 46 PM" src="https://github.com/RoundingWell/care-ops-frontend/assets/35355575/826bef07-fe32-4c4d-acf4-a78cbeb41e5a">

Where program actions have `attributes.behavior: 'conditional'`, but also `relationships._program_flow: null`. Which means the behavior is set to something that's not in the picklist.